modules/buttons/Something_button.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Document</title>
+</head>
+<body>
+    <style>
+      .danger-button{
+        background-color:red;
+        padding:10px;
+        border-width: 0;
+        border-radius:5px;
+        color:#ffffff;
+        font-weight:500;
+        font-size:30px;
+      }
+    </style>
+    <div>
+        <button class="danger-button">
+            <img src="https://res.cloudinary.com/dbcowhqno/image/upload/v1681525774/DPzTPPKeBxFpdAVnKxIg_-transformed-removebg-preview_crqt3h.png" style="height:25px;width:25px;"/>Danger
```

Comment:
what if this sever/url goes down some day? Let's build a powerful and highly efficient system. Use svg for the icon. check recent buttons.